woocommerce/payment-gateway/Blocks/Gateway_Blocks_Handler.php

```diff
@@ -59,7 +59,7 @@ public function handle_blocks_integration() : void {
 			/** @var SV_WC_Payment_Gateway_Plugin $plugin */
 			$plugin = $this->plugin;
 
-			require_once( $plugin->get_framework_path() . '/payment-gateway/Blocks/Gateway_Checkout_Block_Integration.php' );
+			require_once( $plugin->get_payment_gateway_framework_path() . '/payment-gateway/Blocks/Gateway_Checkout_Block_Integration.php' );
```

Comment:
```suggestion
			require_once( $plugin->get_payment_gateway_framework_path() . '/Blocks/Gateway_Checkout_Block_Integration.php' );
```
`/payment-gateway` is included in the `get_payment_gateway_framework_path()` return value